dist/s_evergreen_validate

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+t=__wt.$$
+trap 'rm -f $t' 0 1 2 3 13 15
+
+fast=1
+case "$1" in
+    -F) # Run fast.
+        fast=0
+        shift;;
+    *)
+        if test $# -gt 1 ; then
+            echo "Usage: $0 [-F]"
+            echo "-F only run validation commands if evergreen yml files have been modified"
+        fi
+        break;;
+esac
+
+if [ $(command -v evergreen) ] && [ -f ~/.evergreen.yml ]; then
+    if test $fast -eq 0 ; then
+        # Check the evergreen.yml files for modifications.
+        search=`git diff --name-only $(git merge-base --fork-point develop) | grep -E 'evergreen.*\.yml$'`
```

Comment:
It seems there's an assumption the git command is run for a change against the `develop` branch. Is it the only branch the fast option is targeted for?

Reply:
Correct